hazelcast/src/main/java/com/hazelcast/map/impl/LocalMapStatsProvider.java

```diff
@@ -127,6 +128,12 @@ public LocalMapStatsImpl createLocalMapStats(String mapName) {
             }
         }
 
+        for (String name : statsMap.keySet()) {
```

Comment:
Extra care: runs in calling thread, presumably event handling or proxy API call.